.github/workflows/rotki_docker_publish.yaml

```diff
@@ -0,0 +1,47 @@
+name: Rotki Docker Release
+on:
+  release:
+    types: [published]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  docker:
+    name: Update ${{ github.repository }}:latest on hub.docker.com
+    runs-on: ubuntu-18.04
+    environment: docker
+    env:
+      REGCTL: v0.4.4
+      REGCTL_SUM: 'bafe0048e1b5e8112dfb416e733bb9994b74de6973999a7fb5473ab4125a9eda74df367d5324c2e94726ac0dbfd0445f4b0102f2e43ba5136bedc05b690271c2'
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Rotki Version
+        id: rotki_version
+        run: echo "::set-output name=version::${GITHUB_REF#refs/*/}"
+      - name: Cache regctl
+        uses: actions/cache@v3
+        with:
+          path: regctl-linux-amd64
+          key: ${{ runner.os }}-regctl-${{ env.REGCTL }}
+      - name: Publish ${{ github.repository }}:latest from ${{ steps.rotki_version.outputs.version }}
+        run: |
+          if [[ ! -f "regctl-linux-amd64" ]]; then
+            echo "downloading regctl"
+            curl -LO "https://github.com/regclient/regclient/releases/download/${{ env.REGCTL }}/regctl-linux-amd64"            
```

Comment:
Is a bit ugly that you have to use this to install regctl but seems to be the official way

Reply:
I am not even sure if it is in the official repos, but it seemed the fastest way to get only the specific utility. We don't need everything else in `regclient`.

Also basically every action that uses software that is not pre-installed in the runner pretty much downloads the software from somewhere.

Reply:
Yes, in the repo they also have this as installation method